app/src/main/java/com/quran/labs/androidquran/service/QuranDownloadService.java

```diff
@@ -464,10 +466,11 @@ private boolean downloadFileWrapper(String urlString, String destination,
       }
 
       String url = urlString;
-      if (i > 0) {
-        // let's try http instead of https?
-        if (urlString.contains("quran.com") || urlString.contains("quranicaudio.com")) {
-          url = urlString.replace("https://", "http://");
+      if (fallbackByDefault || i > 0) {
+        if (urlString.contains("android.quran.com")) {
```

Comment:
You could directly use `replace` without if block/check, because with `contains` and `replace` you are doing double look-up anyway.

Reply:
jazakAllah khairan - done